api/swagger.yaml

```diff
@@ -1952,6 +1952,8 @@ definitions:
         x-nullable: false
         type: "integer"
         example: 2
+      Platform:
+        $ref: "#/definitions/OCIPlatform"
```

Comment:
I'm using the existing platform definition, one thing I don't like about this one is that the properties are lowercase...